### PR TITLE
ui(web icon): add avocado svg icon

### PR DESCRIPTION
**Pull request by deborahlow97**

### CODE REVIEW

File: src/components/navigation/Navbar.js [hunk 1]
```diff
@@ -33,6 +35,13 @@ const Navbar = () => {
               style={{ textDecoration: 'none' }}
             >
               Avocadoro
+              <Icon>
+                <img
+                  src={AvocadoroIcon}
+                  alt="AvocadoroToast"
+                  height={20}
```
Comment (brandonyeoxg) [inline src/components/navigation/Navbar.js]: Magic number, prefer using const.

Perhaps you can create a new file `src/const/ui.js` to store all ui const?

**Review by brandonyeoxg — CHANGES_REQUESTED** [review]
Minor nitpick.

File: src/components/navigation/Navbar.js (hunk 1 above)
Comment (deborahlow97) [reply src/components/navigation/Navbar.js]: ok boss

**Review by brandonyeoxg — APPROVED** [review]
LGTM in princinple. In future we will relook into refactoring `const` to not use classes (#51)

**Pull request merged**